carbon-design-system/ibm-security
# feat(ica-update): ica sizes and trending variant

## Pull request 
Added sizes default, lg, and xlg to the ICA component. Added trending variant.

### Affected issues

- Resolves #907 and #908

### Proposed changes

- Styling for large and xlarge ICA
- Added trending icon 
- Added large and xlarge for trending. 
- Improved styling to handle ICAModule and ICAWall cases with the trending arrow.

### Testing instructions

- Go to the ICA story on storybook
- Use the ICA size and trending knobs to test the different sizes. (Should match the styling below).
![image](https://user-images.githubusercontent.com/45407808/109322636-c29f6280-780f-11eb-8a0d-57f250f68528.png)
